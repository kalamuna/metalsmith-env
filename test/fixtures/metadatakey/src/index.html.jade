---
title: Custom metadata key
pretty: true
---
doctype html
html(lang="en")
  head
    title= title
  body
    h1= title
    p This is the environmental variable "npm_package_config_foo":
    code
      pre= env.npm_package_config_foo
