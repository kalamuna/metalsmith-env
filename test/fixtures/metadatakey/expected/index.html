<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Custom metadata key</title>
  </head>
  <body>
    <h1>Custom metadata key</h1>
    <p>This is the environmental variable "npm_package_config_foo":</p><code>
      <pre>bar</pre></code>
  </body>
</html>